### DISCUSSION TITLE
How to implement tray icon in Mac OS?

### DISCUSSION THREAD

**Discussion by wj8400684** [AvaloniaUI/Avalonia#9937] [Q&A]
<img width="722" alt="image" src="https://user-images.githubusercontent.com/39239954/211190633-f499d53d-8b0d-4c50-8570-9ea37e7006da.png">

**Comment by Irasil** [comment]
In the documentation of AvaloniaUI, you can find an example [here](https://docs.avaloniaui.net/docs/reference/controls/tray-icon#example).

**Reply by jpgarza93** [reply]
link is broken

**Reply by Irasil** [reply]
Thanks! The link is fixed now.

**Reply by maxkatz6** [reply]
You can also set `MacOSProperties.IsTemplateIcon="True"` as it's commonly done on macOS (so OS can fill non transparent pixels depending on the theme).
